blockservice/blockservice.go

```diff
@@ -10,6 +10,7 @@ import (
 	"io"
 
 	exchange "github.com/ipfs/go-ipfs/exchange"
+	"github.com/ipfs/go-ipfs/thirdparty/verifcid"
```

Comment:
We should not be putting more stuff in `thirdparty`. If it needs to be re-used across submodules or has interest for other places put it in a different repo, if not, place it in the submodule directly. It complicates extractions afterwards...

Reply:
This is temporary measure, it will be removed as we are able to propagate go-cid.
The temporary measure was applied because we want to release ASAP and have this behaviour fixed in next release.

Reply:
Then place that code as submodules of `blockservice`? That way the temporary measure can go together with the temporary code and be fixed with it when it's extracted. Otherwise I will have to do exactly that.

Reply:
@hsanjuan the reason I want it in thirdparty is explicitly because it needs to be moved to the cid package as soon as possible. But (as you know) we have a bunch of stuff in-flight that prevents us from bubbling up the cid package without other complications, so its going here for now as a "please remove me"

Reply:
@whyrusleeping  sure, I understand the reasons, but it still introduces a blocker for me.

I'm trying to communicate that if it's placed under `blockservice` with a "please-remove-me" note it doesn't become a blocker while solving your problem at the same time.

If you say this will be removed from `thirdparty` right away after the release, OK. I'm just scared that things will take longer though.